src/templates/Gov/Proposals/index.tsx

```diff
@@ -65,14 +72,25 @@ const Proposals = () => {
         <S.AllProposalsContent>
           <S.TitleAndLinkContent>
             <TitleSection image={proposals} title="All Proposals" />
-            <S.LinkForum
-              href="https://gov.kassandra.finance/"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              <span>Discuss the proposals at the Forum</span>
-              <Image src={externalLink} alt="" />
-            </S.LinkForum>
+
+            <S.ButtonWrapper>
+              <S.LinkForum
+                href="https://gov.kassandra.finance/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <span>Discuss the proposals at the Forum</span>
+                <Image src={externalLink} alt="" />
```

Comment:
Add either a `alt` text, or add `aria-hidden="true"` to the image if it's not important for the user with screen reader

```suggestion
                <Image src={externalLink} alt="External Link illustration" />
```

```suggestion
                <Image src={externalLink} alt="" aria-hidden="true" />
```